docs/blog/_posts/2020-03-18-23rd-dotty-milestone-release.md

```diff
@@ -221,7 +221,7 @@ Macro.scala
 import scala.quoted._
 
 inline def showTree(inline x: Any): String = ${ showTreeImpl('x) }
-def showTreeImpl(x: Expr[Any])(using qctx: QuoteContext): Expr[String] =
+def showTreeImpl(x: Expr[Any])(using qctx: Quotes): Expr[String] =
```

Comment:
```suggestion
def showTreeImpl(x: Expr[Any])(using Quotes): Expr[String] =
```

Reply:
Actually, that  one should be `QuoteContext` as it was in a blog post. I fixed it.